platform/darwin/test/MGLLightTest.mm

```diff
@@ -25,7 +25,10 @@ - (void)testProperties {
     {
         mbgl::style::Light light;
         MGLLight *mglLight = [[MGLLight alloc] initWithMBGLLight:&light];
-        XCTAssertEqual(mglLight.anchor, MGLLightAnchorViewport);
+        
+        NSAssert([mglLight.anchor isKindOfClass:[MGLConstantStyleValue class]], @"mglLight.anchor isn’t a MGLConstantStyleValue.");
```

Comment:
@boundsj @1ec5 sorry I didn't see this until recently. I will fix this in a separate branch.